src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -29,7 +29,7 @@ namespace Microsoft.Identity.Client
     public sealed class AcquireTokenForClientParameterBuilder :
         AbstractConfidentialClientAcquireTokenParameterBuilder<AcquireTokenForClientParameterBuilder>
     {
-        private AcquireTokenForClientParameters Parameters { get; } = new AcquireTokenForClientParameters();
+        internal AcquireTokenForClientParameters Parameters { get; } = new AcquireTokenForClientParameters();
```

Comment:
Is this needed?

Reply:
Can you point me to where? @gladjohn

Reply:
@neha-bhargava The `Parameters` property needs to be `internal` because it is accessed in the `AcquireTokenForClientParameterBuilderForResourceProviders` class to set the `AccessTokenHashToRefresh` property. Here is the relevant code snippet:

```cs
public static AcquireTokenForClientParameterBuilder WithAccessTokenSha256ToRefresh( this AcquireTokenForClientParameterBuilder builder, string hash) { if (!string.IsNullOrWhiteSpace(hash)) { builder.Parameters.AccessTokenHashToRefresh = hash; }
```

Making the `Parameters` property `internal` allows it to be accessed within the same assembly, which is necessary for this extension method to function correctly.

Reply:
does this explanation resolve your questions @neha-bhargava, I see you closed it without accepting the reasoning. Is there a better way to do this?